1080i/SkinSettings.xml

```diff
@@ -415,6 +415,14 @@
                                 <onclick>Skin.ToggleSetting(TMDbHelper.DisablePVR)</onclick>
                                 <selected>!Skin.HasSetting(TMDbHelper.DisablePVR)</selected>
                             </include>
+                            <include content="Dialog_Settings_Button" description="Display End Times">
+                                <param name="id" value="7007" />
+                                <param name="control" value="radiobutton" />
+                                <label>$LOCALIZE[31267]</label>
+                                <visible>Container(3).HasFocus(7)</visible>
+                                <onclick>Skin.ToggleSetting(InfoLine.EndTime)</onclick>
+                                <selected>Skin.HasSetting(InfoLine.EndTime)</selected>
+                            </include>
```

Comment:
Can you remove this settings toggle from this PR. The End Time setting has deliberately not been included (for the time being). There are some scenarios where it makes the infoline too long, plus there are a few other issues about when it displays that need to be addressed first.

Also, don't forget to remove the string from strings.po when you remove the setting.

Reply:
Sure. That was one of the things I was missing from AH1, but I understand.